pkgs/applications/editors/mindforger/default.nix

```diff
@@ -1,35 +1,37 @@
-{ mkDerivation, stdenv, fetchurl, qmake, qtbase, qtwebkit }:
+{ mkDerivation
+, cmark-gfm
+, fetchurl
+, qmake
+, qtbase
+, qtwebkit
+, stdenv
+, wrapGAppsHook
+}:
 
 mkDerivation rec {
   pname = "mindforger";
-  version = "1.48.2";
+  version = "1.52.0";
 
   src = fetchurl {
-    url = "https://github.com/dvorka/mindforger/releases/download/1.48.0/mindforger_${version}.tgz";
-    sha256 = "1wlrl8hpjcpnq098l3n2d1gbhbjylaj4z366zvssqvmafr72iyw4";
+    url = "https://github.com/dvorka/mindforger/releases/download/${version}/mindforger_${version}.tgz";
+    sha256 = "1pghsw8kwvjhg3jpmjs0n892h2l0pm0cs6ymi8b23fwk0kfj67rd";
   };
 
-  nativeBuildInputs = [ qmake ] ;
-  buildInputs = [ qtbase qtwebkit ] ;
+  nativeBuildInputs = [ qmake wrapGAppsHook ] ;
+  buildInputs = [ qtbase qtwebkit cmark-gfm ] ;
 
   doCheck = true;
 
-  enableParallelBuilding = true ;
+  prePatch= ''
```

Comment:
Can we move this into postPatch? Otherwise some upstream patches might not apply properly.

Reply:
let me try

Reply:
yeah well spotted, should be fixed now, I retested the GUI manually and all seems fine